static_src/js/.eslintrc

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
I'm all in favor of checking in some linting rules, but if we do, we should probably run `eslint` in ci.

Reply:
ah dang, fat-fingered this commit. do you think I should remove it or add the CI tests? I'm up for either. but if I add lint file, I'll want to update this one so it's... more opinionated. 🤷‍♂️